services/shotgrid_common/ayon_shotgrid_hub/update_from_shotgrid.py

```diff
@@ -342,9 +404,10 @@ def remove_ayon_entity_from_sg_event(
     """
     # for now we are ignoring Task type entities
     # TODO: Handle Task entities
-    if sg_event["entity_type"] == "Task":
-        log.info("Ignoring Task entity.")
-        return
+    # Why?
+    # if sg_event["entity_type"] == "Task":
+    #     log.info("Ignoring Task entity.")
+    #     return
```

Comment:
Indeed, clearer communication was needed. A glitch in the AYON server caused the blockage. @fabiaserra, did it work when you tested it? Which server version are you using?